changelogs/fragments/70904-properly-handle-sshpass-return-codes.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - SSH plugin - Raise an exception when sshpass returns 1-4,6 (https://github.com/ansible/ansible/issues/58113).
```

How about rephrasing this a bit?
```suggestion
  - SSH plugin - Raise an exception when ``sshpass`` return code is in range 1-4 or 6 (https://github.com/ansible/ansible/issues/58113).
```

---

Thank you, corrected.